rails/db/migrate/20180619125128_import_parcels.rb

```diff
@@ -62,7 +62,7 @@ def up
     username = config[Rails.env]["username"] ? "-U #{config[Rails.env]["username"]}" : "-U #{ENV['USER']}"
     port     = config[Rails.env]["port"] ? "-p #{config[Rails.env]["port"]}" : ""
     # pg_restore version must match the DB version
-    system("pg_restore -Fc -a -v -j 8 -t parcels #{host} #{username} #{database} #{port} #{Rails.root}/lib/import/parcels.dump")
+    # system("pg_restore -Fc -a -v -j 8 -t parcels #{host} #{username} #{database} #{port} #{Rails.root}/lib/import/parcels.dump")
```

Comment:
This is commented code

Reply:
Yes, but this time, I want it there as documentation.